src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -13,13 +13,13 @@ if [ ! -d "$__PROJECT_DIR" ]; then
    exit 1
 fi
 
-if [ ! -d "$__TOOLS_DIR" ]; then
-   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 2nd parameter."
    exit 1
 fi
 
-if [ ! -e "$__DOTNET_CMD" ]; then
-   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+if [ ! -d "$__TOOLRUNTIME_DIR" ]; then
```

Comment:
I was thinking that the Tools directory (i.e. `TOOLRUNTIME_DIR`) existing is a prerequisite of invoking init-tools.sh.  The CoreFx init-tools.sh creates this directory when it does `mkdir "$__PROJECT_JSON_PATH"`.

Also, I'm not changing this logic from the way you had it when you created the script. Is there a reason it needs to change?